tutorials/plot_background_statistics.py

```diff
@@ -178,47 +187,47 @@ def plot_t_p(t, p, title, mcc, axes=None):
+# **null distribution** values for each voxel by taking random subsets of
+# samples (subjects), flipping the sign of their data, and recording the
+# absolute value of the resulting statistic (we record the absolute value
+# because we conduct a two-tailed test). The absolute value of the statistic
+# evaluated on the veridical data can then be compared to this distribution,
+# and the p-value is simply the proportion of null distribution values that
+# are smaller.
 #
 # .. note:: In the case where ``n_permutations`` is large enough (or "all") so
 #           that the complete set of unique resampling exchanges can be done
-#           (which is :math:`2^{N_{samp}}-1=1023` for the one-tailed paired
-#           test here, not counting the veridical distribution),
-#           instead of randomly exchanging conditions the null is formed
-#           from using all possible exchanges. This is known as a permutation
-#           test (or exact test) form of a non-parametric resampling test.
+#           (which is :math:`2^{N_{samp}}-1` for a one-tailed and
+#           :math:`2^{N_{samp}-1}-1` for a two-tailed test, not counting the
+#           veridical distribution), instead of randomly exchanging conditions
+#           the null is formed from using all possible exchanges. This is known
+#           as a permutation test (or exact test).
 
 # Here we have to do a bit of gymnastics to get our function to do
 # a permutation test without correcting for multiple comparisons:
 
-# Let's flatten the array for simplicity
-X.shape = (n_subjects, n_src)
+X.shape = (n_subjects, n_src)  # flatten the array for simplicity
 titles.append('Permutation')
 ts.append(np.zeros(width * width))
 ps.append(np.zeros(width * width))
 mccs.append(False)
 for ii in range(n_src):
-    ts[-1][ii], ps[-1][ii] = \
-        permutation_t_test(X[:, [ii]], verbose=True if ii == 0 else False)[:2]
+    ts[-1][ii], ps[-1][ii] = permutation_t_test(X[:, [ii]], verbose=False)[:2]
 plot_t_p(ts[-1], ps[-1], titles[-1], mccs[-1])
 
 ###############################################################################
 # Multiple comparisons
 # --------------------
 # So far, we have done no correction for multiple comparisons. This is
 # potentially problematic for these data because there are
-# :math:`40 \times 40 = 1600` tests being performed. If we just use
-# a threshold ``p < 0.05`` for all of our tests, we would expect many
-# voxels to be declared significant even if there were no true effect.
+# :math:`40 \cdot 40 = 1600` tests being performed. If we use a threshold
+# ``p < 0.05`` for each individual test, we would expect many voxels to be
+# declared significant even if there were no true effect.
 # In other words, we would make many **type I errors** (adapted from
-# `here <https://en.wikipedia.org/wiki/Type_I_and_type_II_errors>`_):
+# `here <https://en.wikipedia.org/w/index.php?title=Type_I_and_type_II_errors#Table_of_error_types>`_):  # noqa
```

Comment:
you should probably use a link as done in `master`, the `# noqa` gets rendered

https://8408-1301584-gh.circle-artifacts.com/0/html/auto_tutorials/plot_background_statistics.html#multiple-comparisons